developer-tools.md

```diff
@@ -435,6 +435,44 @@ Error:(147, 9) value q is not a member of StringContext
         q"""
         ^ 
 ```
+<h4>Debug Spark with SBT and IntelliJ</h4>
```

Comment:
It seems that the content below is just about Remote debug unit tests on SBT **shell** with IntelliJ
The title is too big here.